<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="42" started-at="2019-04-17T20:44:52Z" finished-at="2019-04-17T20:44:52Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="42" started-at="2019-04-17T20:44:52Z" finished-at="2019-04-17T20:44:52Z">
      <class name="com.database.DatabaseDeleteDataFromTable">
        <test-method status="FAIL" signature="deteleDataFromTable()[pri:0, instance:com.database.DatabaseDeleteDataFromTable@598067a5]" name="deteleDataFromTable" duration-ms="14" started-at="2019-04-17T20:44:52Z" finished-at="2019-04-17T20:44:52Z">
          <exception class="java.sql.SQLException">
            <message>
              <![CDATA[No suitable driver found for jdbc:oracle:thin@localhost:1521/pdborc1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.sql.SQLException: No suitable driver found for jdbc:oracle:thin@localhost:1521/pdborc1
at java.sql.DriverManager.getConnection(DriverManager.java:689)
at java.sql.DriverManager.getConnection(DriverManager.java:247)
at com.database.DatabaseDeleteDataFromTable.deteleDataFromTable(DatabaseDeleteDataFromTable.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.sql.SQLException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deteleDataFromTable -->
      </class> <!-- com.database.DatabaseDeleteDataFromTable -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
